### PR TITLE
Support for rsh in DMTCP-3.0

### PR DESCRIPTION
**Pull request by gc00**
This is a port of PR #296 to master (DMTCP-3.0).

### CODE REVIEW

**Comment by gc00** [comment]
@rohgarg, @kapil : @ankitcse07 proposes that we merge this now, and he will test it later at his end.  That seems reasonable to me.  Could one of the two of you review this?

File: src/plugin/ipc/ssh/ssh.cpp [hunk 1]
```diff
@@ -474,10 +512,15 @@ updateCoordHost()
 extern "C" int
 execve(const char *filename, char *const argv[], char *const envp[])
 {
-  if (jalib::Filesystem::BaseName(filename) != "ssh") {
+  if ((jalib::Filesystem::BaseName(filename) != "ssh") &&
```
Comment (rohgarg) [inline src/plugin/ipc/ssh/ssh.cpp]: @gc00: Should we include the changes from PR #455?

**Comment by gc00** [comment]
@rohgarg, I've now added the PR #455 from 2.5 into the PR for this master/3.0 branch.  Thanks for the suggestion.

**Comment by gc00** [comment]
@ankitcse07 , I've now added your 'rsh' support to the DMTCP 3.0 branch.  During the 2017 year, we plan to release the DMTCP 3.0 branch.  Some smaller things will change in 3.0, like the specification for writing plugins.  But it will be easy to upgrade.  And we'll be providing a document to clarify.

**Pull request merged**